digitalfabrik/entitlementcard
# 1514: Use predefined platform git tag

### Short description

Since we don't want to create git tags for all available platforms in a `bump-up step` in some scenarios, we have to pass a particular tags f.e.` native | web | all` depending on what release workflow we trigger.

### Proposed changes



- pass parameters for platform specific deployments
- you can pass string separated by "/" for platforms f.e. `web/native` if you need

### Dependency
depends on this pr to be merged
https://github.com/digitalfabrik/app-toolbelt/pull/14


### Side effects



- none

### Resolved issues



Fixes: #1514

### Testing (dev only)
1. (Before depending pr is merged) You can test it by replacing the following the repo url in `install_app_toolbelt`
with: https://github.com/digitalfabrik/app-toolbelt/archive/refs/heads/bump-up-add-platform-option.tar.gz
2. Choose a workflow that uses `bump_version` with `prepare_delivery: true` like `deliver_beta_backend_administration` but comment out all steps except `bump_version`
3. trigger workflow
4. Check for new git tag
https://github.com/digitalfabrik/entitlementcard/tags
